cmd/nebula-cert/ca.go

```diff
@@ -103,6 +106,20 @@ func ca(args []string, out io.Writer, errOut io.Writer) error {
 		}
 	}
 
+	var passphrase []byte
+	if !*cf.noEncryption {
```

Comment:
I should probably also check if this is a terminal. One option would be to use the SSH terminal module again - https://pkg.go.dev/golang.org/x/crypto/ssh/terminal#IsTerminal

In the name of designing safe APIs, I'd opt for erroring out if `-no-encryption` isn't passed, and the client is not a terminal.